test/e2e/test_organization_function_run.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+from typing import Any, Iterator
+
+import pytest
+
+import layer
+from layer import refresh_login
+from layer.clients import LayerClient
+from layer.contracts.accounts import Account
+from layer.contracts.fabrics import Fabric
+from layer.contracts.projects import Project
+from test.e2e.assertion_utils import E2ETestAsserter
+from test.e2e.conftest import (
+    _cleanup_project,
+    pseudo_random_account_name,
+    pseudo_random_project_name,
+)
+from test.e2e.test_function_dataset_run import (
+    test_remote_run_with_dependent_datasets_succeeds_and_registers_metadata as test_remote_dependent_datasets,
+)
+from test.e2e.test_function_model_run import (
+    test_remote_run_succeeds_and_registers_metadata as test_remote_model_train,
+)
+from test.e2e.test_layer_runtime import (
+    test_dataset_build as test_remote_layer_runtime_dataset_build,
+)
+
+
+# We use a wrapping Test class so we create only one organization
+# until we handle newly acquired permissions (after org creation)
+# better
+class TestOrganization:
+    @pytest.fixture(scope="class")
+    def initialized_organization_account(
+        self, client: LayerClient, request: Any
+    ) -> Iterator[Account]:
+        org_account_name = pseudo_random_account_name(request)
+        account = client.account.create_organization_account(org_account_name)
```

Comment:
Are all users allowed to create an organization account?

Reply:
atm yes, it will evolve later when we add restrictions

Reply:
Not a blocker but one thing to keep in mind so we update the docs when we get there - the current assumption is that anyone can run e2e-tests with their personal account. If the user needs special permissions, then we will need to make it explicit that `e2e-tests` are only for internal use.

We can just add a flag/selector to skip them as well.

Reply:
do you think it is worth adding that flag now to skip them when choosing to?

im thinking when time comes it will be part of the task to do that

Reply:
No, let's leave it for later when it is actually a problem. Just wanted to mention it for future consideration, no need to do this now.